### PR TITLE
Worked on Issue #8

### PR DESCRIPTION
**Pull request by seyipaye**

### CODE REVIEW

File: app/api/user/views.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+
+@router.get("/{user_id}/key")
+async def get_user_api_key(
+    user_id: UUID):
+    
+    # Check Database if User id exist 
+    
+    # If user exist get user APIkey
+    userAPIKey = APIKey(key='3fa85f64-5740-2262-b3fc-2c963f36afa1', user='3fa85f64-5740-2262-b3fc-2c963f36afa1', continent_name='africa')
```
Comment (a11rew) [inline app/api/user/views.py]: I see you have continent_name here, that was a mistake made on the model diagram :sweat_smile:, please take it out

File: app/api/user/schemas.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from uuid import UUID
+from pydantic import BaseModel
+from typing import Optional
+
+
+class APIKey(BaseModel):
+    key: UUID
+    user: UUID
+    continent_name: str | None = None
```
Comment (a11rew) [inline app/api/user/schemas.py]: Here too

**Review by a11rew — CHANGES_REQUESTED** [review]

File: app/api/user/schemas.py (hunk 2 above)
Comment (seyipaye) [reply app/api/user/schemas.py]: Done

File: app/api/user/views.py (hunk 1 above)
Comment (seyipaye) [reply app/api/user/views.py]: Done

**Pull request merged**